rollup.config.js

```diff
@@ -1,25 +1,20 @@
-import babel from 'rollup-plugin-babel'
+import jsx from 'acorn-jsx'
+import typescript from '@rollup/plugin-typescript'
 import commonjs from '@rollup/plugin-commonjs'
 import resolve from '@rollup/plugin-node-resolve'
-import {terser} from 'rollup-plugin-terser'
```

Comment:
TODO: Figure out why JSX syntax was confusing `terser` (it shouldn’t, since `terser` is used in other GitHub TypeScript projects’ `rollup` configs), then restore it.